signing_clients/tests/__init__.py

```diff
@@ -124,21 +129,30 @@ def test_06_line_too_long(self):
         self.assertRaises(ParsingError, Manifest.parse, BROKEN_MANIFEST)
 
     def test_07_wrapping(self):
-        extracted = JarExtractor('signing_clients/tests/test-jar-long-path.zip',
-                                 'signing_clients/tests/test-jar-long-path-signed.jar',
+        extracted = JarExtractor(test_file('test-jar-long-path.zip'),
+                                 test_file('test-jar-long-path-signed.jar'),
                                  omit_signature_sections=False)
         self.assertEqual(str(extracted.manifest), VERY_LONG_MANIFEST)
 
     def test_08_unicode(self):
-        extracted = JarExtractor('signing_clients/tests/test-jar-unicode.zip',
-                                 'signing_clients/tests/test-jar-unicode-signed.jar',
+        extracted = JarExtractor(test_file('test-jar-unicode.zip'),
+                                 test_file('test-jar-unicode-signed.jar'),
                                  omit_signature_sections=False)
         self.assertEqual(str(extracted.manifest), UNICODE_MANIFEST)
 
     def test_09_serial_number_extraction(self):
-        with open('signing_clients/tests/zigbert.test.pkcs7.der', 'r') as f:
+        with open(test_file('zigbert.test.pkcs7.der'), 'r') as f:
             serialno = get_signature_serial_number(f.read())
         # Signature occured on Thursday, January 22nd 2015 at 11:02:22am PST
         # The signing service returns a Python time.time() value multiplied
         # by 1000 to get a (hopefully) truly unique serial number
         self.assertEqual(1421953342960, serialno)
+
+    def test_10_resigning_manifest_exclusions(self):
+        # This zip contains META-INF/manifest.mf, META-INF/zigbert.sf, and
+        # META-INF/zigbert.rsa in addition to the contents of the basic test
+        # archive test-jar.zip
+        extracted = JarExtractor(test_file('test-jar-meta-inf-exclude.zip'),
+                                 test_file('test-jar-meta-inf-exclude-foo.zip'),
```

Comment:
The -foo file gets created by JarExtractor if necessary.  It was originally -signed but that hit the 80 char line limit.  I should probably just leave the parameter out entirely but didn't want to deviate from the other calling conventions in the file to add possible further confusion.